src/vm/comdelegate.cpp

```diff
@@ -2593,15 +2593,6 @@ BOOL COMDelegate::NeedsWrapperDelegate(MethodDesc* pTargetMD)
 {
     LIMITED_METHOD_CONTRACT;
 
-#ifdef _TARGET_ARM_
```

Comment:
@jkotas Without this, I encountered ``UNREACHABLE`` call as ``COMDelegate::GetSecureInvoke`` is currently not implemented.

Could you let me know which changes are required for ``arm\virtualcallstubcpu.hpp`` and JIT, or how to implement ``COMDelegate::GetSecureInvoke``.